app/src/main/java/com/jetbrains/handson/mpp/mobile/AdvancedSearchActivity.kt

```diff
@@ -103,7 +103,7 @@ class AdvancedSearchActivity : AppCompatActivity(), AdvancedSearchContract.View,
         if (!timeChanged){
             return DateTimeTz.nowLocal().format(dateTimeFormat)
         }
-        val newDateTime = DateTime(myYear, myMonth, myDay, myHour, myMinute)
+        val newDateTime = DateTime(year, month, day, hour, minute)
         return newDateTime.format(dateTimeFormat)
```

Comment:
Not a big deal, but you're doing the formatting twice, you could simplify by doing
```kotlin
val dateTime = if (timeChanged) DateTime(year, month, day, hour, minute) else DateTimeTz.nowLocal()
return DateTime.format(dateTimeFormat)
```